mrp_brewing/report/report_stock_raw_materials.xml

```diff
@@ -65,7 +65,7 @@
                                         <span t-att-style="style" t-esc="a.quantity_after_move" />
                                     </td>
                                 </t>
-                                <t t-if="a.location_dest_id.usage in ['production','inventory']">
+                                <t t-if="a.location_dest_id.usage in ['production','inventory','customer']">
                                     <td/>

```

Comment:
I don't understand why location of usage customer appear in this report. This is the raw material report. So the report telling what are the stock variation for the raw material product. I don't think we should add customer usage here